src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -335,20 +337,19 @@ class LSFJobManager extends AbstractLSFJobManager {
 
     @Override
     protected ExecutionResult executeKillJobs(List<BEJobID> jobIDs) {
-        String command = "${LSF_COMMAND_DELETE_JOBS} ${jobIDs*.id.join(" ")}"
+        String command = "${LSF_COMMAND_ENVIRONMENT} ${LSF_COMMAND_DELETE_JOBS} ${jobIDs*.id.join(" ")}"
```

Comment:
Substitute direct field access by method call `environmentString`. I'd prefer that because eventually the number of retries should be configurable, modelling the environment as map is more appropriate e.g. for ProcessBuilder and a similar interface should also be used for any CommandBuilder (which we currently don't have).